dist/index.html

```diff
@@ -5,7 +5,138 @@
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <main class="all-recipes">
+      <h1> Our recipes </h1>
+      <article class="recipe-grid">
+        <div class="col">
+          <div class="individual-recipe">
+            <div class="recipe-image">
+              <img src="https://spoonacular.com/recipeImages/595736-556x370.jpg">
+              <div class="hover-card"> 
+                <h3>Read more...  </h3>
+              </div>               
+            </div>
```

Comment:
Since the divs for the individual recipes represent sections of content (and they will have attached info upon clicking into them) could they be <article> tags instead of divs, for better accessibility to screen readers?

Reply:
Done.